libs/domains/organizations/data-access/src/lib/domains-organizations-data-access.ts

```diff
@@ -18,6 +25,58 @@ export const organizations = createQueryKeys('organizations', {
       return response.data
     },
   }),
+  gitTokens: (organizationdId) => ({
+    queryKey: [organizationdId],
+    async queryFn() {
+      const response = await organizationApi.listOrganizationGitTokens(organizationdId)
+      return response.data.results
+    },
+  }),
+  authProviders: (organizationId) => ({
+    queryKey: [organizationId],
+    async queryFn() {
+      const response = await gitApi.getOrganizationGitProviderAccount(organizationId)
+      return response.data
+    },
+  }),
+  repositories: (organizationId, gitProvider, gitToken) => ({
+    queryKey: [organizationId, gitProvider, gitToken],
+    async queryFn() {
+      if (gitProvider === GitProviderEnum.GITHUB) {
+        const response = await gitApi.getOrganizationGithubRepositories(organizationId, gitToken)
+        return response.data
+      }
+      if (gitProvider === GitProviderEnum.GITLAB) {
+        const response = await gitApi.getOrganizationGitlabRepositories(organizationId, gitToken)
+        return response.data
+      }
+      if (gitProvider === GitProviderEnum.BITBUCKET) {
+        const response = await gitApi.getOrganizationBitbucketRepositories(organizationId, gitToken)
+        return response.data
+      }
+
+      return Promise.all([])
```

Comment:
This is not correct because it's not the same return type. Using strong types as params, this should not be necessary

Reply:
https://github.com/Qovery/console/pull/913/commits/5b763f82917abfa2a67dcdf0b1a9db7ccb4c77d6